multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/OperationInFinalStateHandler.java

```diff
@@ -90,6 +90,9 @@ protected void setOperationState(String processInstanceId, Operation.State state
         Operation operation = operationService.createQuery()
                                               .processId(processInstanceId)
                                               .singleResult();
+        if (isOperationAlreadyFinal(operation)) {
```

Comment:
This change is not related to the problem, but rather fixes the issue where endedAt is set again to set, which is already aborted.